jupyter-book/cellular_structure/annotation.yml

```diff
@@ -3,13 +3,18 @@ name: best_practices_annotation
 channels:
   - conda-forge
   - bioconda
+  - anaconda
 dependencies:
   - conda-forge::python=3.9
-  - conda-forge::scanpy=1.9
+  - conda-forge::scanpy>=1.9.3
   - conda-forge::leidenalg
   - conda-forge::jupyterlab
-  - conda-forge::jupyter_server=1.18.1
+  - conda-forge::jupyter_server>=1.21 # was: =1.18.1
   - conda-forge::umap-learn>=0.5
   - conda-forge::pynndescent
   - bioconda::celltypist
-  - conda-forge::scvi-tools
+  - anaconda::pandas<2.0.0
```

Comment:
Why do we need to pin pandas to less than 2?

Reply:
because it throws a bunch of errors with version 2